core_integrations/yardian

```diff
@@ -0,0 +1 @@
+../core_brands/aeonmatrix
```

Comment:
This should not be using brands, as the parent PR in the core registry also isn't using brands. Please move the images into the `core_brands/aeonmatrix` folder directly.

../Frenck

Reply:
Hey @frenck 
Got it! I assume that you meant move all the images into `core_integration/yardian` and remove `core_brands/aeonmatrix` right?